source/_data/glossary.yml

```diff
@@ -315,22 +315,17 @@
     operating systems. It can be installed standalone (without Home Assistant
     Supervisor), using a Virtual Environment (typically referred to as Home
     Assistant Core installation method) or as a container using Docker
-    (typically referred to as Home Assistant Container installation method).
+    (typically referred to as Home Assistant Container installation method). The
+    [Home Assistant Core installation method is deprecated](https://www.home-assistant.io/blog/2025/05/22/deprecating-core-and-supervised-installation-methods-and-32-bit-systems/). 
```

Comment:
This feels a bit "jumpy", going from Core installation method to Container installation method, then back to that Core installation method is deprecated.

How about:

    Home Assistant Core is the Python program at the heart of Home Assistant. It
    is part of all installation types. It can be installed standalone (without Home Assistant
    Supervisor) as a container using Docker (typically referred to as Home Assistant
    Container installation method). For development it can also be run using a Virtual
    Environment (previously referred as the Home Assistant Core installation method,
    this [installation method is deprecated for production setup](https://www.home-assistant.io/blog/2025/05/22/deprecating-core-and-supervised-installation-methods-and-32-bit-systems/).)